### DISCUSSION TITLE
These 2 latest Win11 updates cause 2 probems

### DISCUSSION THREAD

**Discussion by bathindahelper** [valinet/ExplorerPatcher#3429] [General]
Kb5036591 and Kb5039212 (both or the later one) caused latest version of EP (V22621.3527.65.5 ) to crash 'Explorer.exe' process whenever I right click on the taskbar. And also no longer let me change my windows volume up/down by my mouse wheel when my pointer is in the empty area of my taskbar. And of course, taskbar size also isn't changeable, Open windows Buttons are converted into Icons (but these 2 are perhaps already noted in EP's readme file). Mainly, I can't do without changing volume. This is a must feature, if possible.

I've searched for these issues on EP-Github thru google and also on Github-EP page. But those discussions were a few months old and for older EP versions.

**Comment by pyrates999** [comment]
I was only able to find this update: https://support.microsoft.com/en-us/topic/june-11-2024-kb5039212-os-builds-22621-3737-and-22631-3737-d7f574c0-2b13-48ca-a9fc-a63093b1a2c2

I wasn't able to find the update for this: Kb5036591

Next time you just need to post the windows build number by running winver.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by bathindahelper** [comment]
I'm very sorry that I forgot to post windows version: 22H2 22621.1992 
And another bad is that i mistyped KB number. Correct was: KB5037591

I didn't change any major settings in EP. Moreover, whatever I changed, it was changed almost an year back. And I tried a few main main settings change at that time. No big.

I"m not using anything special. One thing is Open-Shell Start menu customizer. And I also use Ahk. But those may not be interfering to EP. Because it only happens when I install these latest win updates. I'm attaching the screen shot of those 2 updates, just in case there is any typo by me (again :) ).
![photo_2024-06-27_15-57-30](https://github.com/valinet/ExplorerPatcher/assets/49483022/bdc15837-e1f7-4d44-9885-fcf869b14a64)

Also, I"ve uninstalled those 2 windows updates and now my EP is working ok.

**Comment by Amrsatrio** [comment]
> 22621.1992

Are you sure? That version is like almost a year old at this point.

> KB5037591

It is an update for .NET and should not effect EP in any way.

> KB5039212

I don't understand, I daily drive 22631.3737 on my laptop and haven't experienced any crashes or whatsoever... but I've seen that you're using open-shell and mix and matching customization software isn't really a supported scenario here.

**Comment by pyrates999** [comment]
Since you haven't posted screen shots of settings from the EP properties application:

if you enabled the option register as shell extension, disable it by opening up the EP properties application, go to **File Explorer**, and untick **Register as shell extension**.

Here's why: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension
